Formula/fstar.rb

```diff
@@ -36,32 +35,39 @@ def install
                                            "$(DATE_EXEC) '+%Y-%m-%dT%H:%M:%S%z'"
 
     resource("z3").stage do
-      system "python", "scripts/mk_make.py", "--prefix=#{libexec}"
-      system "make", "-C", "build"
-      system "make", "-C", "build", "install"
+      # F* warns if the Z3 git hash doesn't match
+      system "python", "scripts/mk_make.py", "--prefix=#{libexec}", "--githash=1f29cebd4df6"
```

Comment:
you should be able to get this from
```
Utils.popen_read("git", "rev-parse", "--short", "HEAD").chomp
```
or similar

Reply:
I didn't think it was worth the hassle, but I agree it improves maintainability.